worlds/cvcotm/presets.py

```diff
@@ -0,0 +1,159 @@
+from typing import Any, Dict
+
+from Options import Accessibility, ProgressionBalancing
+from .options import IgnoreCleansing, AutoRun, DSSPatch, AlwaysAllowSpeedDash, BreakIronMaidens, BuffRangedFamiliars,\
+    BuffSubWeapons, BuffShooterStrength, ItemDropRandomization, HalveDSSCardsPlaced, Countdown, SubWeaponShuffle,\
+    DisableBattleArenaMPDrain, RequireAllBosses, EarlyDouble, DeathLink, CompletionGoal
+
+all_random_options = {
+    "progression_balancing":         "random",
+    "accessibility":                 "random",
+    "ignore_cleansing":              "random",
+    "auto_run":                      "random",
+    "dss_patch":                     "random",
+    "always_allow_speed_dash":       "random",
+    "break_iron_maidens":            "random",
+    "required_last_keys":            "random",
+    "available_last_keys":           "random",
+    "buff_ranged_familiars":         "random",
+    "buff_sub_weapons":              "random",
+    "buff_shooter_strength":         "random",
+    "item_drop_randomization":       "random",
+    "halve_dss_cards_placed":        "random",
+    "countdown":                     "random",
+    "sub_weapon_shuffle":            "random",
+    "disable_battle_arena_mp_drain": "random",
+    "require_all_bosses":            "random",
+    "early_double":                  "random",
+    "death_link":                    "random",
```

Comment:
Fair warning: having death link being random in the preset may lead to people getting frustrated when they didn't realize that they'd end up with death link on. If you have a way to turn off death link client-side, this is more fine.